ci/cron/src/Main.hs

```diff
@@ -271,18 +272,25 @@ docs = do
 
 download_assets :: FilePath -> GitHubRelease -> IO ()
 download_assets tmp release = do
+    manager <- HTTP.newManager TLS.tlsManagerSettings
     tokens <- Control.Concurrent.QSem.newQSem 20
     Control.Concurrent.Async.forConcurrently_ (map uri $ assets release) (\url ->
         Control.Exception.bracket_
           (Control.Concurrent.QSem.waitQSem tokens)
           (Control.Concurrent.QSem.signalQSem tokens)
           (do
-        shell_ $ unlines ["bash -c '",
-            "set -euo pipefail",
-            "eval \"$(dev-env/bin/dade assist)\"",
-            "cd \"" <> tmp <> "\"",
-            "wget --quiet \"" <> show url <> "\"",
-            "'"]))
+              req' <- HTTP.parseRequest (show url)
+              let req = req' { HTTP.requestHeaders = [("User-Agent", "DAML cron (team-daml-language@digitalasset.com)")] }
```

Comment:
Maybe worth updating that email address but I guess it also doesn’t matter.

Reply:
What would you suggest instead? AFAIK this one still works and will still reach the relevant people.